packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageModule.kt

```diff
@@ -162,6 +162,10 @@ class ExpoImageModule : Module() {
         view.recyclingKey = recyclingKey
       }
 
+      Prop("enableDownscaling") { view: ExpoImageViewWrapper, enableDownscaling: Boolean? ->
```

Comment:
What about `allowDownscaling` or `disableDownscaling`?
For me "enable" means that downscaling will always be used, but in fact it's done only when necessary.